# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../galan/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual gcc-3.4 compile fixes.

  - Rene Rebe <rene@exactcode.de>

--- galan-0.2.14/src/event.c.orig	2004-05-13 20:36:28.045674776 +0200
+++ galan-0.2.14/src/event.c	2004-05-13 20:37:54.848478744 +0200
@@ -55,6 +55,7 @@
 		memcpy( dst->d.array.numbers, src->d.array.numbers, src->d.array.len * sizeof(gdouble) );
 		break;
 	    default:
+		;
 	}
     }
 	
@@ -73,6 +74,7 @@
 		    free( e->d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( e );
     }
@@ -91,6 +93,7 @@
 		    free( evq->e.d.array.numbers );
 		break;
 	    default:
+		;
 	}
 	free( evq );
     }
